### PR TITLE
Only group voter names if the initial vote and the current vote are equal

### PR DESCRIPTION
**Pull request by diedexx**
If a user changed their vote from 3 to 5, you can't tell who it was. This is because both names will be linked to both 5 votes.
image

With this PR, you can
image
image

If both users switch from 3 to 5, both names will be shown on hover.
image

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -138,8 +138,8 @@ <h1 class="finished">Refinement done! 📣 🙌 🎉</h1>
                                 <i :class="['fas','fa-mug-hot']" v-if="vote.currentValue === 'coffee'"></i>
                                 {{ vote.currentValue }}
                                 <div class="vote-names"
-									 v-if="groupedVoterNames && groupedVoterNames[vote.currentValue]">
-                                {{ groupedVoterNames[vote.currentValue] && groupedVoterNames[vote.currentValue].join(', ') }}
+									 v-if="getGroupedVoteNames(vote)">
```
Comment (diedexx) [inline html/index.html:141]: Weirdly enough, Vue's `v-if` considers an empty array as a falsy value

**Comment by moorscode** [comment]
I disliked the grouping of the names to begin with, but as I didn't send a name <-> vote combination it was not possible to determine who voted for which.
I do think it's a nice feature to have now, being used to it :D

File: html/index.html (hunk 1 above)
Comment (moorscode) [reply html/index.html:141]: I had errors about getGroupedVoterNames not being declared, that's why I added it, but a method is always present, so good guy Vue is nice :D

File: src/pokers/pokers.service.ts [hunk 2]
```diff
@@ -311,9 +311,10 @@ export class PokersService {
 				voteCount: votes.length,
 				votes,
 				groupedVoterNames: voted.reduce( ( accumulator, client: Client ) => {
-					const vote                       = this.getRoom( poker ).getCurrentVote( client.id );
-					accumulator[ vote.currentValue ] = accumulator[ vote.currentValue ] || [];
-					accumulator[ vote.currentValue ].push( client.name );
+					const vote                  = this.getRoom( poker ).getCurrentVote( client.id );
+					const voteGroupKey: string  = vote.initialValue + "/" + vote.currentValue;
+					accumulator[ voteGroupKey ] = accumulator[ voteGroupKey ] || [];
+					accumulator[ voteGroupKey ].push( client.name );
```
Comment (moorscode) [inline src/pokers/pokers.service.ts:317]: Nice an simple, I love this solution!

**Pull request merged**